mysql.py

```diff
@@ -641,6 +685,11 @@ def read_callback():
     for key in slave_status:
         dispatch_value('slave', key, slave_status[key], 'gauge')
 
+    if is_ps_enabled(conn) is True:
+        user_connections = fetch_connections_per_account(conn)
+        for key in user_connections:
+            dispatch_value('connection_per_user', key, user_connections[key], 'counter')
```

Comment:
as this query get current connections, wouldn't this value be of gauge type ?

Reply:
Indeed gauge is more appropriate for this metric.